apostrophecms/a3-docs
# Add section on filters to guide->templating

## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*
This PR adds an additional page to the Guide within the 'Templating' section documenting the use of Nunjucks filters. This includes a link out to the documentation for Nunjucks-supplied filters, a brief overview and detailed list of Apostrophe-supplied filters, and details on how to add a custom filter with code example.
Closes PRO-961.

## What are the specific steps to test this change?
1. Build and run the documentation
2. Navigate to Guide->Templating->Template Filters


*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
